randovania/games/am2r/json_data/header.txt

```diff
@@ -223,7 +223,7 @@ Templates
               Charge Beam and Missile-Less Metroid Fights (Advanced)
```

Comment:
Would it be worth it bumping `missile-less metroid fights` up to expert to be in line with dual alphas? Considering shots can get interrupted or you take damage and lose the charge.

Reply:
This is for normal single dodging gammas though. Dual alphas are expert because you got two things going at you. A single gamma is much easier in that regard.

Reply:
Right, then I guess the more appropriate comment would be should dual gammas get a new template? 
![image](https://github.com/randovania/randovania/assets/125271738/3325837d-348a-46a7-b6fc-12eaf1a3edc5)

Reply:
Correct, but that's for the future:tm:

Reply:
Will add this to the "to do" pile I have and close this comment.